www.foia.gov/_sass/modules/_foia_request_form.scss

```diff
@@ -25,19 +25,36 @@
 }
 
 .foia-request-form_inline-progress {
-  color: $color-gray-light;
-  font-size: 1.4rem;
-  font-weight: 700;
+  color: $color-gray;
+  font-size: $base-font-size;
+  margin-bottom: $space-2x;
 }
 
 .foia-request-form_nav-top {
   max-width: 46rem; // Align with input elements
   text-align: right;
+  margin-top: $space-3x;
+  margin-bottom: $space-2x;
 
+  a {
+    @extend .usa-button;
+    @extend .usa-button-outline;
+    &:visited {
+      color: $color-primary-darkest;
+    }
+  }
   // Total hack, if we're following a field-object it is probably a section.
   // Each section already has it's own return to top link, so this is probably
   // a duplicate link at the very bottom of the form.
   .field-object + & {
     display: none;
   }
 }
+
+.foia-request-form_is-required {
+  float: right;
```

Comment:
What is preventing a long label from running into this float?
<img alt="screenshot from 2017-11-06 11-59-43" width="300" src="https://user-images.githubusercontent.com/509703/32461248-12db4a68-c2ea-11e7-8c66-005512e94ef5.png">

Reply:
I could throw a margin on the label to keep that space open. hmm.